### PR TITLE
Update sitemaps.md

### PR DESCRIPTION
**Pull request by m0rgano**
Show an example of displaying an image from an IP camera.

### CODE REVIEW

**Review by ThomDietrich — DISMISSED** [review]
Hey @m0rgano,
your intention is to show the use of refresh? Good idea but this is not so clear from the example. I'd leave out the comment for simplicity but add "e.g. for webcam images" to the `refresh` description. For testing reasons I suggest to use a publicly available webcam link :) there are many for famous public places or even from the ISS. Not sure how to include them though. Maybe you can quickly try.

These are just suggestions, I don't want to hold back the PR. Let us know how you feel about my requests. Best!

**Comment by ghys** [comment]
Note: changed the base branch to `2.5.x` which is the default for now, and will appear on the production website.
`master` is now the branch for openHAB 3.x docs.

@openhab/documentation-maintainers 
Once this is merged to `2.5.x` please remember to merge (for example, cherry-pick the commit) this to `master` as well if appropriate.
See https://github.com/openhab/openhab-docs/issues/1133 for details.

**Comment by Confectrician** [comment]
I am with @ThomDietrich here.
If refresh is the usecase here, we should promote this better.

**Pull request merged**
